packages/react-html/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "react-html",
+  "version": "19.0.0",
+  "private": true,
+  "description": "React package generating embedded HTML markup such as e-mails using Server Components.",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/react.git",
+    "directory": "packages/react-html"
+  },
+  "keywords": [
+    "react"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/facebook/react/issues"
+  },
+  "homepage": "https://react.dev/",
+  "peerDependencies": {
+    "react": "^19.0.0"
```

Should we start off here with the correct approach since renderer and React need to be in lock step?
```suggestion
    "react": "19.0.0"
```

---

Let's change all packages at once if we do that.